src/js/leaflet.storage.js

```diff
@@ -715,6 +745,41 @@ L.Storage.Map.include({
         L.S.fire('ui:start', {data: {html: container}});
     },
 
+    importRaw: function(rawData, type){
+        var importedData = JSON.parse(rawData);
+
+        for (var option in importedData.properties) {
+            this.options[option] = importedData.properties[option];
+        }
+
+        var self = this;
+        importedData.layers.forEach( function (geojson) {
+            var dataLayer = self.createDataLayer();
+            dataLayer.fromUmapGeoJSON(geojson);
+        });
+
+        L.Storage.once('saved', function () {
+            document.location.reload();
```

Comment:
I think that if we only take `editableOptions`, we could get rid of this reload.